src/components/widgets/registration/FormComponent.tsx

```diff
@@ -34,9 +36,14 @@ export const FormComponentBase = ({ className }: RegFormBaseProps) => {
     const [showFirework, setShowFirework] = useState(false);
     const { isMobile } = useScreenDetector();
 
+    const { mutate: sendFormData } = useCreateAppointments();
     const onSubmit = async (data: FieldValues) => {
-        console.log(data);
-        reset();
+        const formData: FormFields = data as FormFields;
```

Comment:
I think if you have problems with types, you should change it in the hook directly. Here type casting looks like 🩼.